doc/pic/hld_fpmsyncd.md

```diff
@@ -475,6 +515,46 @@ Sample of CONFIG DB snippet given below:
         }
     },
 ```
+#### APPL DB Enhancements  
+
+This feature extends the existing NEXT_HOP_GROUP_TABLE, to store next hop group information to be used by one or more routes. Recursive/ecmp routes are represented by referencing a list of nexthop group ids in the nexthop_group field. When this field is used, other fields will not be present.
+
+
+```
+### NEXT_HOP_GROUP_TABLE
+  Stores a list of groups of one or more next hops
+  Status: Mandatory
+    key           = NEXT_HOP_GROUP_TABLE:string ; 
+                    arbitrary string identifying the next hop 
+                    group, as determined by the programming 
+                    application.
+    nexthop       = *prefix,           ; IP addresses separated
+                     “,” (empty indicates no gateway)
+    ifname        = *PORT_TABLE.key,   ; zero or more separated 
```

Comment:
Why just PORT_TABLE? how about portchannel, vlan and sub-interfaces..etc

Reply:
There is no change in this DB Table. This is as is from https://github.com/sonic-net/SONiC/blob/master/doc/ip/next_hop_group_hld.md

Reply:
Do we allow NH interfaces other than PORT_TABLE?

Reply:
Yes, as I understand all possible existing NH interface would be allowed

Reply:
Just mentioning PORT_TABLE alone the schema is weird, please fix it.

Reply:
Changed to "INTF_TABLE" so that all types on interfaces are covered